### PR TITLE
[FIX] Jammy: build using `python3` package

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

**Review by hbrunn — APPROVED** [review]

File: build.yml [hunk 1]
```diff
@@ -160,6 +160,7 @@ docker-targets:
     args:
       from: ubuntu:bionic
       jpeg: libjpeg-turbo8-dev
+      python_package: python3
```
Comment (ashkulz) [inline build.yml:163]:
```suggestion
      python: python3
```
Does just having `python3` work, or you need `python-is-python3` for it to work?

File: docker/Dockerfile.debian [hunk 2]
```diff
@@ -15,6 +16,6 @@ RUN apt-get update && apt-get install -y -q --no-install-recommends \
     libx11-dev \
     libxext-dev \
     libxrender-dev \
-    python \
+    $python_package \
```
Comment (ashkulz) [inline docker/Dockerfile.debian:19]:
```suggestion
    $python \
```

File: docker/Dockerfile.debian [hunk 3]
```diff
@@ -2,6 +2,7 @@ ARG  from
 FROM ${from}
 
 ARG  jpeg=libjpeg-dev
+ARG  python_package=python
```
Comment (ashkulz) [inline docker/Dockerfile.debian:5]:
```suggestion
ARG  python=python
```

**Review by ashkulz — COMMENTED** [review]
Thanks for the quick PR, @StefanRijnhart! I assume you have tried it locally and it works?

File: docker/Dockerfile.debian (hunk 2 above)
Comment (StefanRijnhart) [reply docker/Dockerfile.debian:19]: I considered this, but I found it ambiguous. `python` could refer to Python package, or Python binary etc. Are you sure you want me to rename the variable?

Comment (ashkulz) [reply docker/Dockerfile.debian:19]: I think we use `jpeg` the same way, and it's understood that it's the package.

File: build.yml (hunk 1 above)
Comment (ashkulz) [reply build.yml:163]: Perfect, thanks!

**Comment by ashkulz** [comment]
I've started a [new run](https://github.com/wkhtmltopdf/packaging/actions/runs/2548711281) which should complete in a few hours.

**Pull request merged**